### ISSUE TITLE
Error on libray

### ISSUE THREAD

**Issue by camilomalagon** [OPM/opm-common#283]
The Simulator stop working around a week ago.

I try to reinstall it and I get an error during installation.
It says that:

The following packages have unmet dependencies:
libopm-output1: Depends: libecl1 but it is not going to be installed
libopm-parser1: Depends: libecl1 but it is not going to be installed
libopm-simulators1: Depends: libecl1 but it is not going to be installed
libopm-simulators1-bin: Depends: libecl1 but it is not going to be installed
E. Unmet dependencies. Try 'apt-get -f install' with no packages (or specify a solution)

when using that line "apt-get -f install"

It states that
dpkg: Error processing archive /var/cache/apt/archives/libecl1_2017.10-rfinal-1-xenial_amd64.deb (--unpack)

Can you help me by telling what went wrong or tell me how to delete Flow in order to try a reinstall from zero.

Thanks

**Comment by atgeirr** [comment]
> The Simulator stop working around a week ago.

Some change must have been made to your system for this to happen, no OPM software performs automatic updating at this point.



> Can you help me by telling what went wrong or tell me how to delete Flow in order to try a reinstall from zero.

You should probably start with "apt-get remove libopm-simulators" to uninstall the binaries (verify by trying to run "flow" and being told that it does not exist). Apart from that I am afraid I do not know (I am not on Ubuntu as my daily system).

**Comment by akva2** [comment]
try this:

```
sudo dpkg -r libert.ecl
sudo rm  /var/cache/apt/archives/libecl1_2017.10-rfinal-1-xenial_amd64.deb 
sudo apt-get -f install
```

**Comment by camilomalagon** [comment]
What I mean is that when Ubuntu wanted to install an update, it can not.  And then it reported the issue to be.
"The package system is broken. "Check if you are using a third party repository or further more use apt-get install -f to solve the issue.""
The report says that the problem is the transaction: libopm-simulators1

And when you run that command line the problem is in the libecl1.  as I said before.

I run the command : apt-get remove libopm-simulators and I get the following.
"Note, selecting 'libopm-simulators1' instead of 'libopm-simulators'
and then the usual "you have un met dependencies bla bla bla (as reproted in the first post)

Therefore seems to me that for some reason the software is looking for files that have a number 1 in addition to the normal names.

Any second and finnal though about this ? before I just reinstall ubuntu?

**Comment by akva2** [comment]
no, that's not an issue, that's just the package series numbering and is expected. libecl is an alias for libecl1 etc.

**Comment by camilomalagon** [comment]
I quit trying to run flow on the desktop and I went to my cluster, where I had it running correctly and now is not working there either...
after entering the command 
flow SPEcmn.DATA
I get
Symbol lookup error: /lib64/libopmparser.so.2017: undefined symbol: -ZN3ERT11smspec_nodeC1E19ecl_smspec_var_typeRKSsS3

Can you send me the command to update the simulator to the new release?

Thank you

**Comment by akva2** [comment]
1) clean out old version
sudo apt-get remove libert.ecl* libopm-*
2) install new version
sudo apt-get install libopm-simulators-bin

**Comment by camilomalagon** [comment]
Hi,  
I did as you suggested above and it solved the issue on the laptop. Ubuntu was able to update and now the simulator is running.

sudo dpkg -r libert.ecl1
sudo rm  /var/cache/apt/archives/libecl1_2017.10-rfinal-1-xenial_amd64.deb 
sudo apt-get -f install

I can not make changes on the Cluster although.  

should I give the Cluster admin the following code so he can do the to update?

sudo apt-get remove libert.ecl* libopm-*
sudo apt-get install libopm-simulators-bin

**Comment by akva2** [comment]
yes. i'm sorry for the trouble, as i said i should have marked the two packages (libert.ecl, libecl) to be in conflict to make the error more obvious.

**Comment by andlaus** [comment]
I tested the packages on my Xenial machine, and they worked fine for me. possibly there is an old libert library lurking on your system? if you did not install ERT manually at some point, you should be able to get rid of it via `sudo apt-get autoremove`...

**Comment by camilomalagon** [comment]
Hi thank you so much for the support. now it is running well in the Ubuntu.

My admin. in the Cluster tell me that the system there is a RedHat therefore the commands are not the same to fix it.

Basically the error is the same:
camilomn-gorina2:~/Sim/RunFlow/SPEcmn1$ flow SPEcmn.DATA                        
flow: symbol lookup error: /lib64/libopmparser.so.2017: undefined symbol: _ZN3ERT11smspec_nodeC1E19ecl_smspec_var_typeRKSsS3_

Can you give me the commands to fix it in Red Hat please.

Cheers

**Comment by akva2** [comment]
yum remove libert.ecl* ert.ecl* libopm* opm*
yum install opm-simulators-bin
